gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
Is there something that is holding us back from using 6.7? It is the version we use in freshly generated game projects, and for me it looks like it works without problems on the main repo, too.

Reply:
Just playing safe, I feel confident upgrading to 5.4.1 because it's in theory fully backwards compatible with 5.4 and the basic tests I ran worked. I agree it would be ideal upgrading to 6.7 at some point but the potential of breaking something is higher there and we would require more tests. Do you want to fo all in?

Reply:
Yes, I use 6.7 for quite some time in my projects without problems and we don't do the full build with it anyway.

Reply:
I've upgraded Gradle to `6.7.1` (the Gradle docs recommend to always use the latest patch version) and haven't found any issue. This allows us to upgrade the Android Gradle plugin, I've chosen 4.0.2 instead of 4.1 due to not being supported on IDEA 2020.2. I'll run some more tests on iOS and Android, it'd be great if somebody could test GWT stuff.